uber/tests/models/test_badge_funcs.py

```diff
@@ -15,10 +15,11 @@ def session(request):
 
 @pytest.fixture(autouse=True)
 def teardown_range_check(request):
-    def _check_range():
-        with Session() as session:
-            check_ranges(session)
-    request.addfinalizer(_check_range)
```

Comment:
Oh, derp. That's really odd - I re-ran the unit tests, which should have caught the if statement being wrong, but didn't. I'll fix and redo the PR real quick.